src/components/shared/Dropdown.css

```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
   background-color: var(--search-overlays-semitransparent);
 }
 
+.dropdown li img.prettyPrint {
+  -webkit-mask: url(/images/prettyPrint.svg) no-repeat;
```

Comment:
we have autoprefixer, so you should be able to use just mask :)

Reply:
Changed.